spec/lib/utils/job-utils/racadm-tool-spec.js

```diff
@@ -339,6 +358,12 @@ describe("racadm-tool", function() {
                 this.sandbox.restore();
             });
 
+            it('should throw Error if no pathFile is found', function(){
+                expect( function() {
+                    return instance.setBiosConfig('192.168.188.103', 'admin', 'admin');
+                }).throw(Error, 'Can not find file path required for set BIOS configuration');
```

Comment:
it's better to expect the setBiosConfig return a rejected Promise not throw error